rs_client/staging_client.py

```diff
@@ -0,0 +1,274 @@
+
+        staging_body = {  # pylint: disable=line-too-long
+            "version": "0.2.0",
+            "id": "staging",
+            "title": {"en": "Staging"},
+            "description": {
+                "en": "A process that takes an external STAC ItemCollection, asynchronously download"
+                "its assets into the RS catalog bucket and creates the corresponding STAC items in the RS catalog.",
+            },
+            "jobControlOptions": ["async-execute"],
+            "keywords": ["stac", "staging"],
+            "links": [
+                {
+                    "type": "text/html",
+                    "rel": "about",
+                    "title": "documentation",
+                    "href": "https://home.rs-python.eu/rs-documentation/rs-server"
+                    "/docs/doc/users/functionalities/#staging",
+                    "hreflang": "en-US",
+                },
+            ],
+            "inputs": {
+                "collection": {
+                    "title": "Target collection",
+                    "description": "The target collection identifier in the RS catalog",
+                    "id": out_coll_name,
+                    "schema": {"type": "string"},
+                    "minOccurs": 1,
+                    "maxOccurs": 1,
+                },
+                "items": stac_item_collection.model_dump(mode="json"),
+                "provider": "cadip",
+            },
+            "outputs": {
+                "result": {
+                    "title": "Output STAC items",
+                    "id": "some_output_id",
+                    "description": "The staged STAC ItemCollection",
+                    "schema": "false",
+                    "minOccurs": 1,
+                    "maxOccurs": 1,
+                },
+            },
+        }
+
+        try:
+            post_response = self.http_session.post(
+                url=self.href_execute_process,
+                json=staging_body,
+                timeout=TIMEOUT,
+                **self.apikey_headers,
+            )
+
+            if post_response.status_code != 200:
+                self.logger.warning(f"Staging response status code: {post_response.status_code}")
+
+            # Monitor the running job
+            resp = json.loads(post_response.content)
+            pprint.PrettyPrinter(indent=4).pprint(resp)
+
+            self.job_id = resp["status"]["started"]
+            print(f"\nJob ID = {self.job_id}\n")
+            if post_response.status_code == 200 and self.job_id:
+                self.logger.info(f"Staging job {self.job_id} successfully launched !")
+
+        except KeyError as e:
+            self.logger.exception(f"Could not launch the staging - response doesn't have the following key: {e}")
+            return post_response.status_code, None
+
+        return post_response.status_code, self.job_id
+
+    def get_jobs(self):
+        """Method to get running jobs"""
+        return self.http_session.get(
```

Comment:
@vprivat-ads what return type do you expect for this endpoint ? A list of job identifiers for example ?